app/build.gradle

```diff
@@ -94,13 +94,17 @@ android {
     }
     lintOptions {
         disable 'MissingTranslation'
+        disable 'InvalidPackage'
```

Comment:
Why?

Reply:
If this cannot be avoided then consider using `warning 'InvalidPackage'`
https://stackoverflow.com/questions/21827004/gradle-dagger-lint-ignore-by-package#comment58243384_21985399

Reply:
Updated. Another solution is: https://github.com/getsentry/sentry-java/issues/571#issuecomment-378295015